build.sbt

```diff
@@ -1,3 +1,6 @@
+import _root_.sbtrelease.ReleasePlugin.autoImport.ReleaseStep
+import _root_.sbtrelease.ReleasePlugin.autoImport._
+import _root_.sbtrelease.ReleaseStateTransformations._
```

Comment:
Minor: You can replace all three imports with `import ReleaseStateTransformations._` ReleasePlugin and ReleaseStep are automatically imported by that.